core/arch/arm/pta/sub.mk

```diff
@@ -9,6 +9,7 @@ endif
 srcs-$(CFG_WITH_STATS) += stats.c
 srcs-$(CFG_TA_GPROF_SUPPORT) += gprof.c
 srcs-$(CFG_TEE_BENCHMARK) += benchmark.c
+srcs-$(CFG_SDP_PA_UTIL) += sdp_pta.c
```

Comment:
`CFG_SDP_PA_UTIL` could be rename to `CFG_SDP_PTA`.
We could even  use `CFG_SECURE_DATA_PATH` for this.

Reply:
Since not everyone using secure data path needs this pTA, I'd rather use a dedicated CFG (altough it is really not a big piece of code). I'm all for `CFG_SDP_PTA`.

Reply:
Sorry for skipping change it, I will fix it by using `CFG_SDP_PTA` in next version.